src/frontend/src/utils/statusList/common.ts

```diff
@@ -1,7 +1,28 @@
 import { IPiaForm } from '../../types/interfaces/pia-form.interface';
-import { statusList } from './statusList';
 import { getUserRole } from '../../utils/user';
 import { PiaStatuses } from '../../constant/constant';
+import {
+  completeStatus,
+  cpoReviewStatus,
+  editInProgressStatus,
+  finalReviewStatus,
+  incompleteStatus,
+  mpoReviewStatus,
+  pendingCompletionStatus,
+} from './statuses/index';
+import { StatusList } from './types';
+
+export const statusList = (pia: IPiaForm | null): StatusList => {
```

Comment:
I didn't mean adding statusList to common. I think you should add a function to common that helps you access the correct modal when using `from`, or refactor it so you don't need `from`.

Reply:
I see that you have removed from in favour of the submitModal property. Not sure if we want to keep statusList here or move it back to it's own file.

Reply:
> I see that you have removed from in favour of the submitModal property. Not sure if we want to keep statusList here or move it back to it's own file.

I think we could keep managing StatusList on the common. But if we consider breaking them into smaller modules for better maintainability, it is better to move it back.

Reply:
@BradyMitch I reverted it moving back to separate statuslist file because it is better to maintain on smaller modules